api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -6,7 +6,9 @@
 #include "slint-esp.h"
 #include "slint-platform.h"
 #include "esp_lcd_panel_ops.h"
-#include "esp_lcd_panel_rgb.h"
+#if SOC_LCD_RGB_SUPPORTED && ESP_IDF_VERSION_MAJOR >= 5
```

Comment:
SOC_LCD_RGB_SUPPORTED is defined indirectly from esp_lcd_panel_rgb.h via "soc/soc_caps.h". I'm not sure if the other header we include includes this.

Reply:
Maybe with __has_include ?

Reply:
Good idea, can't harm :)

Reply:
Ohh, or do you mean for `esp_lcd_panel_rgb.h` and then make everything else conditional to that?